meta-resin-common/conf/distro/include/resin-os.inc

```diff
@@ -4,6 +4,25 @@ require conf/distro/poky.conf
 DISTRO = "resin-os"
 DISTRO_NAME = "Resin OS"
 DISTRO_VERSION = "1.1.4"
+HOSTOS_VERSION = "${DISTRO_VERSION}"
+python () {
+    ''' Set HOSTOS_VERSION from board VERSION if available '''
+
+    # Find board VERSION location based on poky meta layer
+    bblayers = d.getVar('BBLAYERS', True).split()
+    poky = [layer for layer in bblayers if layer.endswith('poky/meta')]
+    if not poky:
+        return
+    board_version_path = os.path.join(poky[0], '../../../', 'VERSION')
```

Comment:
Version is the one in resin-board. Has no connection with the one from supervisor.